### PR TITLE
Add Webex

### PR DESCRIPTION
**Pull request by askmrsinh**
This adds the Webex ([https://web.webex.com](https://web.webex.com)) channel.

References:
 - https://developer.webex.com/docs/api/v1/messages/create-a-message
 - https://developer.webex.com/docs/basics#formatting-messages
 - https://developer.webex.com/docs/api/basics#message-attachments

### CODE REVIEW

**Comment by askmrsinh** [comment]
@atymic

**Comment by askmrsinh** [comment]
Can I please get a code review? @freekmurze @mpociot

**Comment by askmrsinh** [comment]
@atymic 
Is this project no longer maintained?
I am asking since I would like to add another channel but I don't see any recent activity / new channels.

**Comment by askmrsinh** [comment]
@mpociot @freekmurze - I wanted to check if anyone of you could please help me with a review and/or with setting up of a Webex channel repo?

File: .github/workflows/coverage.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    name: Unit test coverage
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
```
Comment (atymic) [inline .github/workflows/coverage.yml]: 8.2 please

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  - push
+  - pull_request
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        php: [7.2, 7.3, 7.4, 8.0]
```
Comment (atymic) [inline .github/workflows/tests.yml]: update these to 8.1 +

File: .scrutinizer.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+filter:
```
Comment (atymic) [inline .scrutinizer.yml]: can drop this

File: composer.json [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+            "homepage": "https://askmrsinh.com",
+            "role": "Developer"
+        }
+    ],
+    "require": {
+        "php": ">=7.2",
+        "ext-json": "*",
+        "egulias/email-validator": "*",
+        "guzzlehttp/guzzle": "^6.3 || ^7.0",
+        "illuminate/notifications": "~5.5 || ~6.0 || ~7.0 || ~8.0 || ~9.0",
```
Comment (atymic) [inline composer.json]: only supported laravel vers needed

File: composer.json [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+    "authors": [
+        {
+            "name": "Ashesh Singh",
+            "email": "ask@mrsinh.com",
+            "homepage": "https://askmrsinh.com",
+            "role": "Developer"
+        }
+    ],
+    "require": {
+        "php": ">=7.2",
```
Comment (atymic) [inline composer.json]: only supported php vers

File: composer.json [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+        "ext-json": "*",
+        "egulias/email-validator": "*",
+        "guzzlehttp/guzzle": "^6.3 || ^7.0",
+        "illuminate/notifications": "~5.5 || ~6.0 || ~7.0 || ~8.0 || ~9.0",
+        "illuminate/support": "~5.5 || ~6.0 || ~7.0 || ~8.0 || ~9.0"
+    },
+    "require-dev": {
+        "mockery/mockery": "^1.0",
+        "phpunit/phpunit": "^8.0|^9.0"
```
Comment (atymic) [inline composer.json]: latest phpunit please

**Review by atymic — CHANGES_REQUESTED** [review]
Looks good! minor changes requested. 
Will see if i can remember how to release a new channel.

File: .github/workflows/coverage.yml (hunk 1 above)
Comment (askmrsinh) [reply .github/workflows/coverage.yml]: Update to >=PHP 8.1 + Laravel 10

File: .scrutinizer.yml (hunk 3 above)
Comment (askmrsinh) [reply .scrutinizer.yml]: Done!

File: composer.json (hunk 4 above)
Comment (askmrsinh) [reply composer.json]: Update to Laravel 10

File: composer.json (hunk 5 above)
Comment (askmrsinh) [reply composer.json]: Update to >=PHP 8.1

File: composer.json (hunk 6 above)
Comment (askmrsinh) [reply composer.json]: Using newer PHPUnit 10, compatible with PHP 8.1

File: .github/workflows/tests.yml (hunk 2 above)
Comment (askmrsinh) [reply .github/workflows/tests.yml]: Done!

**Comment by atymic** [comment]
Repo created, please push code there:
https://github.com/laravel-notification-channels/Webex

Then i will release on packagist

**Review by atymic — APPROVED** [review]

**Comment by askmrsinh** [comment]
@atymic  
~~I am unable to fork the repo (screenshot below), or push to it.~~
Also, should it be named `webex` (all lowercase), which is more in line with other channel repos?

**Comment by askmrsinh** [comment]
Sorry, I missed the Github invite to the repo earlier. I have pushed to the new repository.
I strongly feel it should be renamed to `laravel-notification-channels/webex`; I don't have access to do that myself.

Currently trying to get Scrutinizer setup on the new repo:
image

**Comment by askmrsinh** [comment]
Moved to GitHub Actions. Closing.

**Pull request closed without merging**